packages/token-bridge-sdk/src/hooks/useArbTokenBridge.ts

```diff
@@ -436,12 +436,14 @@ export const useArbTokenBridge = (
     erc20L1Address,
     amount,
     l1Signer,
-    txLifecycle
+    txLifecycle,
+    destinationAddress
   }: {
     erc20L1Address: string
     amount: BigNumber
     l1Signer: Signer
     txLifecycle?: L1ContractCallTransactionLifecycle
+    destinationAddress?: string
```

Comment:
<img width="582" alt="Screenshot 2022-12-07 at 18 27 35" src="https://user-images.githubusercontent.com/47932951/206248861-10868837-7f5f-4afd-96b9-3369defe2b72.png">

In case of sending deposits defaults are already applied in sdk if provided `destinationAddress` is undefined. For withdrawals I have defaulted to `walletAddress` if there is no `destinationAddress`